app/scripts/comp/settings-store.js

```diff
@@ -7,36 +7,45 @@ const Logger = require('../util/logger');
 const logger = new Logger('settings');
 
 const SettingsStore = {
+
     fileName: function(key) {
-        return key + '.json';
+        return `${key}.json`;
     },
 
-    load: function(key) {
+    load: function(key, callback) {
         try {
             if (Launcher) {
                 const settingsFile = Launcher.getUserDataPath(this.fileName(key));
-                if (Launcher.fileExists(settingsFile)) {
-                    return JSON.parse(Launcher.readFile(settingsFile, 'utf8'));
-                }
+                Launcher.fileExists(settingsFile, exists => {
+                    if (exists) {
+                        Launcher.readFile(settingsFile, data => {
+                            const contents = typeof data === 'string' ? data : String.fromCharCode.apply(null, data);
```

Comment:
This line there because of different return type here:
https://github.com/keeweb/keeweb/blob/develop/app/scripts/comp/launcher.js#L63

But in cordova It always returns Buffer.
maybe we can add `readFileAsText`?